client/packages/portal-pages/src/pages/view-page/ViewPage.tsx

```diff
@@ -4,11 +4,13 @@ import { StyledBackgroundSection, StyledMain } from '../common-styles/Styles';
 import { StyledContentSection, StyledContentWrapper } from './ViewPage.Styles';
 import { PasePageHeader } from './ViewPageHeader';
 
+import { ViewContextAvailableDialog } from './ViewContextAvailableDialog';
+import { Typography } from '@equinor/eds-core-react';
+
 export const ViewPage = (): JSX.Element => {
 	const { isLoading, currentView } = useViewController();
 
 	if (isLoading) return <FullPageLoading detail="Loading view" />;
-
```

Comment:
Will you make a component of it later?

Reply:
I have a component for it so ill change it :)